src/app/store/subnet/types/base.ts

```diff
@@ -120,3 +125,14 @@ export type SubnetState = GenericState<Subnet, APIError> & {
   eventErrors: EventError<Subnet, APIError, SubnetMeta.PK>[];
   statuses: SubnetStatuses;
 };
+
+export type StaticDHCPLease = {
+  id: number;
+  comment?: string | null;
+  ip_address: string;
+  mac_address: string;
+  interface?: string;
+  node?: SimpleNode;
+  usage?: string | null;
```

Comment:
Let's make sure all optional keys have `| null`, (`node?: SimpleNode | null;`) and not optional key (`node?: SimpleNode;`). With an optional key it can be unclear whether the property was omitted intentionally or by mistake. `null` states explicitly that a value has been intentionally omitted.

Reply:
Adding `null` to the node type throws a lint error when creating the DHCPLease factory, could you please advise on this?

Reply:
You could try resolving it with `() => simpleNode() ?? null,` although there is likely a better way to do this.

Reply:
e.g. something like:
```tsx
export const staticDHCPLease = define<StaticDHCPLease>({
  node: derive<StaticDHCPLease, SimpleNode | null>(simpleNode),
  ```

Reply:
Thanks, this works